src/components/AssigneeIcon.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div aria-hidden="true" class="assignee-icon">
+    <span
+      class="photo avatar"
+      v-if="!assignee.photo"
+      v-bind:style="{ backgroundColor: getAvatarColor(assignee.name) }"
+    >
+      <div
+        class="Avatar AvatarPhoto AvatarPhoto--small AvatarPhoto--color2"
+        role="img"
+        :aria-label="assignee.name"
+      >
+        <div aria-hidden="true">{{ getAssigneeInitials(assignee.name) }}</div>
+      </div>
+    </span>
+    <img
+      class="photo"
+      v-if="assignee.photo"
+      :src="assignee.photo?.image_21x21"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import { Assignee } from "@/types/asana";
+import { computed, defineComponent, PropType } from "vue";
+import avatarColors from "../utils/avatar-colors";
+
+export default defineComponent({
+  props: {
+    assignee: {
+      type: Object as PropType<Assignee>,
+      required: true,
+    },
+  },
+  setup(props) {
+    const getAssigneeInitials = (name: string | null | undefined) => {
+      if (name) {
+        const nameSplit = name.split(" ");
+        const firstInitial = nameSplit[0][0];
+        const lastInitial = nameSplit[nameSplit.length - 1][0];
+        return nameSplit.length > 1 ? firstInitial + lastInitial : firstInitial;
+      }
+    };
+
+    // Assigns a color to the icon based on the assignee's name, similar to Asana's behavior.
+    // Hashing function based on https://stackoverflow.com/questions/7616461/generate-a-hash-from-string-in-javascript.
+    const getAvatarColor = (name: string | null | undefined) => {
```

Comment:
Let's move this function to the avatar-colors file.

The colours don't add much value without this function (arguably this function is the most important part). So I think it makes sense to colocate the two (and only export this function)